backend/app/requests/caching.py

```diff
@@ -0,0 +1,81 @@
+"""Methods to cache adequacy and representative points responses."""
+import json
+import logging
+import os
+import six
+
+import ujson
+
+from backend.config import config
+
+logger = logging.getLogger(__name__)
+
+CACHE_DIRECTORY = config.get('cached_result_directory')
```

Comment:
Can we maybe have a "cache" key in the config with
{enable: true, directory: 'blablabala'} ?

Reply:
Done.